### DISCUSSION TITLE
Feature Request: Presence indicators for DMs/People

### DISCUSSION THREAD

**Discussion by vertigo220** [element-hq/element-meta#928] [Ideas]
Supposedly there are presence indicators for people in rooms (though I can't seem to see any), but AFAICT there are none for individual people in DMs, i.e. those under the "People" category in the left sidebar. In other messengers, including Discord, Steam, and XMPP, each friend/user shows whether they're online, busy, offline, etc, but there doesn't appear to be any such indicators in Element. This is a feature I use a lot, so I can see when friends are available, and it's one of the bigger drawbacks I'm finding switching over from Discord.

**Comment by aaronraimist** [comment]
@vertigo220 what server are you on? Most public servers, such as matrix.org, have the presence feature disabled.

**Reply by vertigo220** [reply]
I'm on matrix.org, though planning to move off of it soon. So hopefully the one I chose has them enabled, then. Shouldn't this be something left up to the user rather than the server?

**Reply by ghost** [reply]
@vertigo220 Presence is a very heavy feature that is currently not viable for use on large homeservers like matrix.org, therefore it is disabled, otherwise the server would be way slower.

**Reply by vertigo220** [reply]
Ah, ok. That makes sense, though surprising it would add much overhead. Thanks.
